src/App.js

```diff
@@ -7,7 +7,8 @@ import { getFirestore } from 'firebase/firestore';
 import AppContext from './components/AppContext';
 import config from './config';
 
-function App() {
+
+export default function App() {
```

Comment:
As per discussion, we should stick to one or the other of these styles.

Reply:
Yes, just missed that file in reverting it back, nice catch.